NEWS.md

```diff
@@ -50,6 +52,12 @@ assessment. (#1960)
     
     - The `description` field was added to `event()`. It can be used to provide
     a description of the event in plain language.
+    
+- `derive_var_atoxgr_dir()` was enhanced (#1859):
+  
+  - can now select `atoxgr_criteria_daids` in argument `meta_criteria` to create `ATOXGRL` and `ATOXGRH` based on [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)
```

Comment:
```suggestion
  -  Can now select `atoxgr_criteria_daids` in argument `meta_criteria` to create `ATOXGRL` and `ATOXGRH` based on [Division of AIDS (DAIDS) Table for Grading the Severity of Adult and Pediatric Adverse Events](https://rsc.niaid.nih.gov/sites/default/files/daidsgradingcorrectedv21.pdf)
```

Reply:
I think if we add `()` then it will auto-link to the reference page

Reply:
Don't think we should add the '()' as thats  not allowed in the argument, just the text `atoxgr_criteria_daids`. I capitalised the first letter at start of sentence, ie `Can`